data/compliance_profiles/CentOS/7/disa_stig.json

```diff
@@ -260,6 +260,32 @@
         ],
         "value": true
       },
+      "auditd::config::audit_profiles::simp::audit_session_files": {
+        "identifiers": [
+          "SRG-OS-000392-GPOS-00172",
+          "SRG-OS-000470-GPOS-00214",
+          "SRG-OS-000473-GPOS-00218",
+          "RHEL-07-030610"
+        ],
+        "oval-ids": [
+          "xccdf_org:ssgproject:content_rule_audit_rules_login_events_faillock"
+        ],
+        "notes": "The SIMP profile applies a base set of audit rules that adhere to guidance from several sources",
+        "value": true
+      },
+      "auditd::config::audit_profiles::simp::audit_session_files_tag": {
+        "identifiers": [
+          "SRG-OS-000392-GPOS-00172",
+          "SRG-OS-000470-GPOS-00214",
+          "SRG-OS-000473-GPOS-00218",
+          "RHEL-07-030610"
```

Comment:
From U_Red_Hat_Enterprise_Linux_7_V1R4_STIG.zip, I think this should be RHEL-07-03600